Sources/minify/src/JS.php

```diff
@@ -15,7 +15,7 @@
 class JS extends Minify
 {
     /**
-     * Var-matching regex based on http://stackoverflow.com/a/9337047/802993.
+     * Var-matching regex based on https://stackoverflow.com/a/9337047/802993.
```

Comment:
https://meta.stackexchange.com/questions/223728/permalink-short-urls-lose-https
There is a bug report on it.